src/main/java/teammates/ui/webapi/action/SearchAccountsAction.java

```diff
@@ -267,7 +267,7 @@ private String generateResultUrl(StudentAttributes student, String fsName) {
             if (ib.getGoogleId() != null) {
                 ib.setHomePageLink(Config.getFrontEndAppUrl(Const.WebPageURIs.INSTRUCTOR_HOME_PAGE)
                         .withUserId(instructor.googleId)
-                        .toAbsoluteString());
+                        .toString());
```

Comment:
Reason for this?

Reply:
As the instructor manage account link and the student manage account link and homepage link all use relative url, I believe it would be more consistent to use a relative url for the instructor homepage link. 

It is also more convenient for testing as you would not be redirected to localhost:4200 when accessing from localhost:8080